README.md

```diff
@@ -1,12 +1,19 @@
 # KAMI
 [![Build Status](https://travis-ci.com/zeroeightysix/KAMI.svg?branch=master)](https://travis-ci.com/zeroeightysix/KAMI)
+[![Maintainability](https://api.codeclimate.com/v1/badges/81228111eb82b7ca391e/maintainability)](https://codeclimate.com/github/zeroeightysix/KAMI/maintainability)
 [![Issues](https://img.shields.io/github/issues/zeroeightysix/kami.svg)](https://github.com/zeroeightysix/kami/issues)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](http://discord.gg/9hvwgeg)
 
-A minecraft utility mod for anarchy servers.
+## A minecraft utility mod for anarchy servers.
 
 Development has slowed down considerably. KAMI will likely no longer be updated. Pull requests are still welcome.
 
+See [forgehax](https://github.com/fr1kin/forgehax) for a more polished equivalent. Some features in KAMI may be based on those of forgehax, as I sometimes used it as reference.
+
+Please note Baritone is no longer included. Download the standalone jar [from here](https://github.com/cabaletta/baritone/releases).
+
+This is by no means a finished project and is still in rapid development.
+
```

Comment:
This is `Development has slowed down considerably. KAMI will likely no longer be updated. Pull requests are still welcome.` on master - should stay that way.

Reply:
This was reverted somewhere along the line, fixed.